### ISSUE TITLE
Missing Release

### ISSUE THREAD

**Issue by Rhatalin** [SirTrollsalot/Desktopcommands#4]
I was looking at your projects when i noticed there was no release. I'm sure you just forgot to add it so i wanted to notify you to prevent further disgrace.

![MissingRelease](https://user-images.githubusercontent.com/17675227/66349130-791f5080-e958-11e9-8b06-38e44c7ac0df.PNG)

**Comment by sedlak477** [comment]
Yes i noticed this myself, I was very confused at first. After some minutes of collecting some fleeting thoughts it came to mind that it might be a mistake, it must be. So I plead you to add a release to your project. I cannot stress enough the confusion and disaster this mishap created for me.

lg
